packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-confirm-sign-in/amplify-confirm-sign-in.component.ts

```diff
@@ -18,36 +14,24 @@ const logger = new Logger('ConfirmSignIn');
   selector: 'amplify-confirm-sign-in',
   templateUrl: './amplify-confirm-sign-in.component.html',
 })
-export class AmplifyConfirmSignInComponent implements OnInit, OnDestroy {
+export class AmplifyConfirmSignInComponent implements OnInit {
   @HostBinding('attr.data-amplify-authenticator-confirmsignin') dataAttr = '';
 
-  public remoteError = '';
-  public isPending = false;
-
-  private authSubscription: Subscription;
-
   // translated texts
   public headerText: string;
   public confirmText = translate('Confirm');
   public backToSignInText = translate('Back to Sign In');
 
-  constructor(private authenticator: AuthenticatorService) {}
+  constructor(public authenticator: AuthenticatorService) {}
 
   ngOnInit(): void {
-    this.authSubscription = this.authenticator.subscribe((state) => {
-      this.onStateUpdate(state);
-    });
```

Comment:
We no longer subscribe to the state machine because `AuthenticatorService` now maintains and provides relevant contexts like `error` or `isPending`.